update_database.py

```diff
@@ -0,0 +1,48 @@
+import database_auth
+import requests
+import json
+from datetime import date
+from datetime import datetime
+from mysql.connector import Error
+
+connection = database_auth.connection()
+
+api_url = 'https://api.tfl.gov.uk/bikepoint'
+
+try:
```

Comment:
this should be wrapped inside functions - and the rest of the code as well. It increases readability and re-usability significantly.